tests/test_main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+from app.main import get_human_age
+
+
+def test_right_convert_in_human_age():
+    assert get_human_age(100, 100) == [21, 17]  # checking the differences between large numbers
```

Comment:
You should create a **separate test** for each class of equivalents. For example:
```
def test_should_return_zero_when_age_is_equal_to_zero():
     assert get_human_age(0, 0) == [0, 0]
     assert get_human_age(14, 14) == [0, 0]


def test_should_return_one_when_age_is_between_15_and_24():
     assert get_human_age(15, 15) == [1, 1]
     assert get_human_age(24, 24) == [1, 1]
```